src/components/select/option/index.js

```diff
@@ -0,0 +1 @@
+export { default } from './option.component';
```

Comment:
We're exposing two ways of using option.

- `import {Option} from 'carbon-react/lib/components/select`
- `import Option from 'carbon-react/lib/components/select/option`

We should have only one way to do this.

Reply:
Unresolved.

Reply:
Unresolved.

Reply:
index file removed